Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM kernsuite/base:6
+FROM kernsuite/base:7
 
 ################################
 # install latest masters
 ################################
-RUN echo "deb-src http://ppa.launchpad.net/kernsuite/kern-5/ubuntu bionic main" > /etc/apt/sources.list.d/kernsuite-ubuntu-kern-5-bionic.list
+RUN echo "deb-src http://ppa.launchpad.net/kernsuite/kern-7/ubuntu bionic main" > /etc/apt/sources.list.d/kernsuite-ubuntu-kern-7-bionic.list
+RUN add-apt-repository ppa:cartavis-team/carta
+RUN add-apt-repository ppa:saiarcot895/chromium-beta
 RUN apt-get update
 RUN docker-apt-install --no-install-recommends \
```

Comment:
the `--no-install-recommends` flag suppresses the installation of `casacore-data` which is recommended for `CARTA`.
Otherwise, we can add it to the list of dependencies below to get it from _kern-suite_.